frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -22,10 +23,21 @@ import { VMConsoleFirehose } from './vm-console';
 import { VMDetailsFirehose } from './vm-details';
 import { vmMenuActionsCreator } from './menu-actions';
 import { VMDashboard } from './vm-dashboard';
-import { TEMPLATE_TYPE_LABEL, TEMPLATE_TYPE_VM, VM_DETAIL_ENVIRONMENT } from '../../constants/vm';
+import {
+  TEMPLATE_TYPE_LABEL,
+  TEMPLATE_TYPE_VM,
+  VM_DETAIL_ENVIRONMENT,
+  VM_DETAIL_SNAPSHOTS,
+} from '../../constants/vm';
 import { VMEnvironmentFirehose } from './vm-environment/vm-environment-page';
 import { VMNics } from '../vm-nics';
 import { PendingChangesWarningFirehose } from './pending-changes-warning';
+import { VMSnapshotsPage } from '../vm-snapshots/vm-snapshots';
+import {
```

Comment:
nitpick: This should be above the kubevirt package imports. Of course, we can catch it in a follow-up if you don't have any other changes to make.